assets/scripts/outputs.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+set -e
+
+# set filename for storing variables and values
+output_file="output.txt"
+
+# set header and footer strings
+header="###pipeline-summary-start###"
+footer="###pipeline-summary-end###"
+
+# check if file exists, if not create it
+if [ ! -f "$output_file" ]; then
```

Comment:
You don't actually need the if statement - you can just touch it all the time - if it exists, all it will do is update the time stamp.